src/components/Avatar/Avatar.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { UserImage } from '@ynput/ayon-react-components'
```

Comment:
Make sure to use UserImage from /src/components/UserImage.jsx and then only pass the name prop

Reply:
I did after our talk today. 👍